Tests/ApolloTests/RequestCreatorTests.swift

```diff
@@ -283,4 +284,135 @@ Bravo file content.
       self.checkString(stringToCompare, includes: endString)
     }
   }
+
+  func testRequestBodyWithApolloRequestCreator() {
+    let query = HeroNameQuery()
+    let req = apolloRequestCreator.requestBody(for: query, sendOperationIdentifiers: false)
+
+    XCTAssertEqual(query.queryDocument, req["query"] as? String)
+  }
+
+  // MARK: - Custom request creator tests
+
+  func testSingleFileWithCustomRequestCreator() throws {
+    let alphaFileUrl = self.fileURLForFile(named: "a", extension: "txt")
+
+    let alphaFile = GraphQLFile(fieldName: "upload",
+                                originalName: "a.txt",
+                                mimeType: "text/plain",
+                                fileURL: alphaFileUrl)
+
+    let data = try customRequestCreator.requestMultipartFormData(
+      for: HeroNameQuery(),
+      files: [alphaFile!],
```

Comment:
Probably worth `guard let` ing this on creation, since a force-unwrap failing in a test causes the whole test suite to crash rather than failing the single test.

Reply:
This is just copy/paste from other code made by you 🙈
Should I remove all force unwrap in this class?

Reply:
🤦‍♀️Nah I'll fix it in a follow-up PR.

Reply:
Sowwy 🙈

Reply:
Haha, I'm the moron who did it first, no worries